src/css/controls/imports/controlbar.less

```diff
@@ -12,13 +16,11 @@
     border-radius: 0;
     background-size: auto;
     box-shadow: none;
-    height: @mobile-touch-target * 1.5;
-    padding: @slider-fixed-height 0 0;
+    max-height: 72px;
+    //padding: @slider-fixed-height 0 0;
```

Comment:
The control bar needs left+right padding so that the slider knob doesn't go off the player and to make it usable on touch devices.

Before:
<img width="439" alt="screen shot 2017-07-31 at 2 54 46 pm" src="https://user-images.githubusercontent.com/4603952/28793131-0809f170-7600-11e7-8a8e-eb3539052a49.png">

After adding 14px:
<img width="442" alt="screen shot 2017-07-31 at 2 54 26 pm" src="https://user-images.githubusercontent.com/4603952/28793140-0d91ce1a-7600-11e7-9371-b33fdec07a5d.png">